File: 27-Remove-Element.py
class Solution:
    def removeElement(self, nums: List[int], val: int) -> int:
        start, end = 0, len(nums) - 1
        while start <= end:
            if nums[start] == val:
                nums[end], nums[start] = nums[start], nums[end]
                end -= 1
            else:
                start += 1
        return start

File: go/27-Remove-Element.go
func removeElement(nums []int, val int) int {
	left, right := 0, len(nums)-1

	for left <= right {
		if nums[left] == val {
			nums[left], nums[right] = nums[right], nums[left]
			right--
		} else {
			left++
		}
	}

	return left
}